src/webots/control/WbControlledWorld.cpp

```diff
@@ -71,7 +71,7 @@ WbControlledWorld::WbControlledWorld(WbProtoList *protos, WbTokenizer *tokenizer
   QFile file(WbStandardPaths::webotsTmpPath() + "WEBOTS_SERVER");
   if (file.open(QIODevice::WriteOnly)) {
     QTextStream stream(&file);
-    stream << mServer->fullServerName().toUtf8() << endl;
+    stream << mServer->fullServerName().toUtf8() << '\n';
```

Comment:
Note that at the other occurrence of this issue you fixed it that way:
```suggestion
    stream << mServer->fullServerName().toUtf8() << Qt::endl;;
```

Reply:
I had problems with compiling that on Linux. So, I believe it's simpler to use `\n` everywhere.